<?xml version="1.0"?>

<launch>
  <arg name="namespace" default="/vigir_walk_controller" />
  <group ns="$(arg namespace)">
    <!-- start walk controller -->
    <node name="walk_controller" pkg="vigir_walk_control" type="walk_controller_node" respawn="true" output="screen" />

    <!-- load plugin descriptions from YAML file to parameter server -->
    <rosparam file="$(find vigir_walk_control)/config/plugin_descriptions.yaml" command="load" />
  </group>
</launch>
